### DISCUSSION TITLE
Extract 'state' value per cell per gene

### DISCUSSION THREAD

**Discussion by wenhu0701** [welch-lab/MultiVelo#40] [Q&A]
Hello,

Thanks for making this package and it's very helpful!
Now I'm trying to retrieve the 'State' value of each gene and each cell (basically it's the result showing as mv.dynamic_plot), Could you please show me how to do it?

Thanks!
Wen

**Comment by danielee0707** [comment]
Hi Wen. It should be in the output anndata's layers field, for example adata_result.layers['fit_state']. If you want to get the values for several genes, you can extract these by using `adata_result[:,target_gene_names].layers['fit_state']`.
